.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - id: boards
         run: echo "::set-output name=board_matrix::$(ls ./boards --indicator-style=none | jq -ncR '[inputs]')"
       - id: features
-        run: echo "::set-output name=feature_matrix::["reconfigurable-system-pins","usart-spi-host-without-select"]"
+        run: echo "::set-output name=feature_matrix::[\"reconfigurable-system-pins\",\"usart-spi-host-without-select\"]"
```

Comment:
Having the features hard coded like this isn't pretty, but I have no idea how to parse the actual list of features and not catch the internal-only features.

Reply:
There is probably a better way to check different feature permutations (and maybe streamline the CI further), but we can delay that until we have more feature gates. Ideally they will be de-coupled, so we could just check with `--all-features`, but I don't know if we can separate the PACs from this.

Reply:
There a cargo extensions crate called [cargo-hack](https://github.com/taiki-e/cargo-hack) that can do feature permutations.

Raised #42 about this.